deskpro/deskpro-admin-frontend
# fix/8024-table-boolean-filters

Allow `YES_NO` operator to handle `BOOL` filters

Issue:
![image](https://user-images.githubusercontent.com/32859498/72893757-ebab1500-3d2a-11ea-86c9-34a91aa326d4.png)

Steps to review:
 * Open agents table page
 * Click on `Filter` button at `TableActions`
 * Select `admin_common.col.can_admin` property and `Yes` as item
 * Click `Apply filter` button
 * Check filtered table
